game.conf

```diff
@@ -1,3 +1,3 @@
 name = Minetest Game
 author = Minetest
-description = Bundled by default with Minetest, and aims to be lightweight, moddable, and fairly playable without mods.
+description = A basic exploration, mining, crafting and building sandbox game with no NPCs, monsters or animals. Minetest Game is usually used with mods added, and many mods are available for this game. Reliably maintained by Minetest Engine core developers.
```

Comment:
(removed a comma)
```suggestion
description = A basic exploration, mining, crafting and building sandbox game with no NPCs, monsters or animals. Minetest Game is usually used with mods added and many mods are available for this game. Reliably maintained by Minetest Engine core developers.
```

Reply:
That is fine for me, want me too update it or are you offering to alter on merge?

Reply:
Oh yeah I can just do so myself...